src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -137,24 +139,24 @@ The Hero currently merges three times a day: At 9 AM (morning), 12 PM (noon), an
 
 To start a release:
 
-1. Create a branch based off develop Github Desktop by clicking **Current Branch** in the top header, clicking **New Branch** in the dropdown, and then selecting **Develop**.
-2. Name the branch following this pattern: `daily-release/mm-dd-yy-morning/noon/evening`. Here's an example: `daily-release/10-27-21-morning`.
-3. Push your changes by clicking **Push Origin** in GitHub Desktop.
+1. Create a branch of `develop` using Github Desktop by clicking **Current Branch**. in the top header, clicking **New Branch** in the dropdown, and then selecting **Develop**.
```

Comment:
This step 1 is all about making sure you're on the `develop` branch *before* you create a new branch. I think the original didn't make that very clear. 

I'd prefer something like:

"It's important to create a new release branch off of the `develop` branch. Before you create a new branch, make sure you're on develop." and then continue with the specific steps.

It's worth mentioning the Keyboard Maestro "Release" macro that automates this process.